google-cloud-spanner/src/main/java/com/google/cloud/spanner/TransactionRunnerImpl.java

```diff
@@ -1052,6 +1114,13 @@ private <T> T runInternal(final TransactionCallable<T> txCallable) {
                     .putAll(TraceUtil.getExceptionAnnotations(e))
                     .put("Attempt", AttributeValue.longAttributeValue(attempt.longValue()))
                     .build());
+            OpenTelemetryTraceUtil.addEvent(
+                openTelemetrySpan,
+                "Transaction Attempt Failed in Commit",
+                Attributes.builder()
+                    .put("Status", ((SpannerException) e).getErrorCode().toString())
```

Comment:
nit: the cast to `SpannerException` should not be necessary here. It is already declared as a `SpannerException`.

Reply:
Done